src/virtac/virtac_server.py

```diff
@@ -408,13 +426,20 @@ def _create_mirror_records(self, mirror_csv: str):
                     try:
                         mirror_type = MIRROR_TYPES[MirrorType(line["mirror_type"])]
                         if mirror_type == MIRROR_TYPES[MirrorType.BASIC]:
+                            mirror_type = cast(type[MonitorPV], mirror_type)
                             # MonitorPV requires a list of str rather than a list of PV
                             output_pv = mirror_type(
                                 out_pv_name,
                                 record_data,
                                 input_pv_names,
                             )
                         else:
+                            mirror_type = cast(
+                                type[InversionPV]
```

Comment:
Why are you using type[InversionPV] instead of InversionPV? Am I missing something?

Reply:
I realised my mistake, since mirror_type refers to the class.